jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -69,6 +69,10 @@
   /** Used to generate the User-Agent header and history metadata. */
   private static final String TOOL_NAME = "jib-gradle-plugin";
 
+  /** Used to generate the User-Agent header and history metadata and verify versions. */
+  static final String TOOL_VERSION =
```

Comment:
private static

So Gradle doesn't have something like `PluginDescriptor`?

Reply:
Yeah I don't think so, I tried looking :\

Reply:
The one option is to search the build classpath for jib-gradle-plugin and extract the version from there. But that's basically the same as doing this.